en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/D/Disrupt_Sense.md

```diff
@@ -25,7 +26,19 @@ _2nd-level necromancy_
 **Components:** V \
 **Duration:** 1 minute
 
-Until the spell ends, one creature you can pinpoint is [blinded](#Conditions_blinded) or [deafened](#Conditions_deafened) (your choice) unless it succeeds on a Constitution saving throw.
+Until the spell ends, one creature you can pinpoint is [deprived](#Conditions_deprived) of one sense unless it succeeds on a Constitution saving throw.
+Choose the sense to disrupt from the following list:
+
+- sight
+- hearing
+- smell
+- [parasense (air pressure)](#Exploration_Environment_parasense)
+- [parasense (echolocation)](#Exploration_Environment_parasense)
```

Comment:
If we do decide to differentiate echolocation (as discussed in my comment on _Animate Objects_), this may have to be modified as well.